invenio_rdm_records/services/services.py

```diff
@@ -22,12 +22,20 @@ class RDMRecordService(RecordDraftService):
     default_config = config.RDMRecordServiceConfig
 
 
-class RDMUserRecordsService(RecordService):
+class RDMUserRecordsService(RecordDraftService):
     """Bibliographic user records service."""
 
     config_name = "RDM_RECORDS_BIBLIOGRAPHIC_USER_RECORDS_SERVICE_CONFIG"
     default_config = config.RDMUserRecordsServiceConfig
 
+    def user_records_search(self, identity, params=None, links_config=None,
+                            es_preference=None, **kwargs):
+        """Search for records matching the querystring."""
+        index_or_alias = self.record_cls.user_records_index._name
```

Comment:
On user records search we use the `user_records_index` provided in the `CommonFieldsMixin` and we point to the common alias of drafts and records.